meson.build

```diff
@@ -1214,6 +1214,8 @@ if want_blkid != 'false' and not skip_deps
 
         conf.set10('HAVE_BLKID_PROBE_SET_HINT',
                    have and cc.has_function('blkid_probe_set_hint', dependencies : libblkid))
+        conf.set10('HAVE_BLKID_SUBLKS_FSINFO',
+                   have and cc.has_header_symbol('blkid.h', 'BLKID_SUBLKS_FSINFO', dependencies : libblkid))
```

Comment:
hm, BLKID_SUBLKS_FSINFO is a macro anyway, so you check for existance of a macro and then set a macro as result? why bother? why not check for the macro in the code directly, via `#ifdef BLKID_SUBLKS_FSINFO`?

what am I missing?

Reply:
Yes, you absolutely right. No reason for that. Haven't questioned myself on why to do it this way. Changed to just `#ifdef BLKID_SUBLKS_FSINFO`, revert changes in `meson.build`.